fnet/cli/init.py

```diff
@@ -39,12 +44,11 @@ def save_default_train_options(path_save: str) -> None:
         Save path for default training options json.
 
     """
-    if os.path.exists(path_save):
-        print('Training options file already exists:', path_save)
+    path_save = Path(path_save)
```

Comment:
You may want to make this `Path(path_save).expanduser().resolve()`. It will make it so people can put relative paths and also include their home directory like `~/Desktop/trained_fnet_model/`

Reply:
Cool! I'll look to add this type of functionality when I address #128.